src/adaptors/graphql/index.js

```diff
@@ -19,9 +26,11 @@ export default function createAdaptor(endpoint, schema) {
   const reducer = combineReducers(reducers);
   const store = createStore(reducer);
 
+
   return {
-    resolve(queries, args, isDataLoaded) {
```

Comment:
I think ```isDataLoaded``` is really just a requirement of the GraphQL implementation. It's the responsibility of any adaptor to resolve all the required data properties at once, and since this is used to do a partial fetch it belongs in the adaptor.